src/pages/dashboard/monitor/components/ActiveChart/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+
+import { Statistic } from 'antd';
+import { MiniArea } from '../Charts';
+import styles from './index.less';
+
+function fixedZero(val: number) {
+  return val * 1 < 10 ? `0${val}` : val;
+}
+
+function getActiveData() {
+  const activeData = [];
+  for (let i = 0; i < 24; i += 1) {
+    activeData.push({
+      x: `${fixedZero(i)}:00`,
+      y: Math.floor(Math.random() * 200) + i * 50,
+    });
+  }
+  return activeData;
+}
+
+export default class ActiveChart extends Component {
+  state = {
+    activeData: getActiveData(),
+  };
+
+  timer: number | undefined = undefined;
+
+  requestRef: number | undefined = undefined;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'requestRef' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)